### DISCUSSION TITLE
Welcome to Project-86 Discussions!

### DISCUSSION THREAD

**Discussion by Taliayaya** [Taliayaya/Project-86#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zh4iYuJia** [comment]
I'm a 86 fan in China and I'm very excited about there's 86lovers making fps game about 86
I've played the demo recently and I want to report something to u guys

1.the respawn point of player is underground which i have to use graphook to move up and stand normally
2.lowe can't detect player from its left or right, 
3.the running sound of juggernaut may broken when pressing shift
My English is not good so theere might be some problems with my expression

In this demo i've seen the high mobility of juggernaut, which made me think of an another high mobility shooter(doom), I'm wondering if juggernauts with melee weapon like undertaker can fight like doom :)

**Reply by Taliayaya** [reply]
Helloo!
Thank you for your remarks, I appreciate it. I'm glad that 86 fans from across the world likes this project!

1. For the respawn point, I thought I fixed it but it's probably too low again
2. The Lowe has a range so that it can see from in front or on the sides until a certain angle. So then it can see you anymore. It will be improved in the next version
3. The running sound at high speed is just a template, in the next version I'll add the good sound and it should be fixed.

Concerning the melee weapon, it is considered but not possible for now as we don't have an actual model to work with it and that it is depend of animations.

Thank you again for your message. Your english is all fine, mine is not perfect either hehe.

**Comment by koukkari1234** [comment]
Hi big fan of 86 and was really looking forward to playing this but for some reason moving my mouse doesn't change where the juggernaut looks only control I have over aiming is movement and adjusting it up or down with the scroll wheel

**Reply by Taliayaya** [reply]
Hello!
This is the first time such issue is shared.
Would you mind sharing what kind of configuration you are using so we can further investigate?

OS: e.g. Windows 11
Game version: e.g. v1.11.0-alpha
Other specs...

If it happens on any mission or just a specific one. If you see any errors, you may share as well the log file located at `%USERPROFILE%\AppData\LocalLow\Project-86-Community\Project-86\Player.log` file so I can see what happened.

Thanks,
Tali

**Reply by koukkari1234** [reply]
Hello I'm not sure what caused it but it fixed itself when I deleted and redownloaded the game this is a very fun game indeed keep up the good work
